### DISCUSSION TITLE
Cannot use `#` as part of the file name in rest api

### DISCUSSION THREAD

**Discussion by binsee** [community/community#4391] [Other Feature Feedback, Questions, & Ideas] [answered]
When I created a file using rest api, I found that the name of the file to be manipulated cannot contain the `#` symbol.
This symbol will be treated as a terminator, and the file name will lose the `#` symbol and the following part.

For example: `hello#world.md` will be treated as `hello`
Screenshot:
![image](https://user-images.githubusercontent.com/5285894/123781718-26917f00-d907-11eb-8249-3693c01c6eab.png)

I did not find relevant instructions in the rest api documentation. Is this a bug?

**Comment by YaFou** [comment, marked as answer]
`#` should be encoded into `%23` in an URL

**Reply by binsee** [reply]
Indeed, it has been resolved, thank you very much.
